### PR TITLE
Support for python3

### PR DESCRIPTION
**Pull request by trojkat**

### CODE REVIEW

File: querystringsafe_base64.py [hunk 1]
```diff
@@ -37,7 +39,9 @@ def encode(to_encode):
         string - like base64, except characters ['+', '/', '='] are
         replaced with ['-', '_', '.'] consequently
     """
-    return urlsafe_b64encode(bytes(to_encode)).replace('=', '.')
+    if PY2:
+        return urlsafe_b64encode(to_encode).replace('=', '.')
+    return urlsafe_b64encode(to_encode).replace(b'=', b'.').decode()
```
Comment (mmaslowskicc) [inline querystringsafe_base64.py]: `'=' == b'='` on Python 2, only `decode` is conditional (if we really want Unicode base64).

File: querystringsafe_base64.py [hunk 2]
```diff
@@ -57,4 +61,6 @@ def decode(encoded):
     # If unicode comes in, strange TypeError is thrown from unicode.translate
     # because it differs from str.translate. Base64 and querystringsafe_base64
     # are ascii-compatible anyway so convert to str.
-    return urlsafe_b64decode(str(encoded).replace('.', '='))
+    if PY2:
+        return urlsafe_b64decode(str(encoded).replace('.', '='))
+    return urlsafe_b64decode(bytes(encoded, 'ascii').replace(b'.', b'='))
```
Comment (mmaslowskicc) [inline querystringsafe_base64.py:64]: So only Unicode strings work as input on Python 3?

File: tests/test_encode_decode.py [hunk 3]
```diff
@@ -108,8 +108,6 @@ def test_querystringsafe_base64_decode_handles_unicode():
 
     # urlsafe_b64decode can handle str but breaks on unicode.
```
Comment (mmaslowskicc) [inline tests/test_encode_decode.py]: Update the comment.

**Comment by tomislater** [comment]
:+1:

**Pull request merged**